lib/thrift/parser/file_group.ex

```diff
@@ -175,12 +175,27 @@ defmodule Thrift.Parser.FileGroup do
   end
 
   def dest_module(file_group, Constant) do
-    # for constants the name of the module is
-    #   <Namespace>.<filename>
-    initial_file = file_group.initial_file
-    base = Path.basename(initial_file, Path.extname(initial_file))
-    name = String.to_atom(base <> "." <> Macro.camelize(base))
-    dest_module(file_group, name)
+    # Default to naming the constants module after the namespaced, camelized
+    # basename of its file. For foo.thrift, this would be `foo.Foo`.
```

Comment:
I'm surprised that El/ixir allows a module name that doesn't start with a capital. 
Or, am I misreading something